x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/timeline/toggle_column.spec.ts

```diff
@@ -47,9 +47,9 @@ describe('toggle column in timeline', () => {
       'exist'
     );
 
-    cy.get(
-      `[data-test-subj="timeline"] [data-test-subj="toggle-field-${timestampField}"]`
-    ).uncheck({ force: true });
+    cy.get(`[data-test-subj="timeline"] [data-test-subj="toggle-field-${timestampField}"]`, {
+      timeout: DEFAULT_TIMEOUT,
```

Comment:
@MadameSheema should we move DEFAULT_TIMEOUT into `cypress.json`, or is the intention to only use it for slow actions?

Reply:
@rylnd timeouts are a little bit tricky in Cypress. These are the ones that we can set in the configuration (https://docs.cypress.io/guides/references/configuration.html#Timeouts). If we extract that time out to the json file, as `defaultCommandTimeout` that timeout is only for the element in the DOM but when we put the timeout like that, we are also setting that timeout to the next action in this case the uncheck so in this case is used with intention. Nevertheless as now are tests are faster I'll revisit all of them in the refactor I'm working on. But for now for having the tests working with the latest master changes I would say that is ok :)